README.md

```diff
@@ -1,2 +1,141 @@
-# micro-rdk-esp32-sensor-examples
-Example micro-rdk sensors for ESP32 intrinsics
+# Viam Micro-RDK ESP32 Sensor Modules Example
+
+## (In)stability Notice
+
+> **Warning** The Viam Micro-RDK is currently in beta.
+
+## Overview
+
+This project in this repository was generated from the [Micro-RDK
+Module
+Template](https://github.com/viamrobotics/micro-rdk-module-template),
+and demonstrates how to produce modular resources for the Micro-RDK.
+
+## Usage
+
+### Setup
+
+If you don't yet have a Micro-RDK robot project, please create one by
+following the [Micro-RDK Development
+Setup](https://docs.viam.com/installation/prepare/microcontrollers/development-setup/)
+instructions.
+
+### Installation
+
+To add this module to a robot project, just add this repository to the
+`[dependencies]` section of the `Cargo.toml` file in the robot
+project:
+
+
+``` diff
+diff --git a/Cargo.toml b/Cargo.toml
+index 79fbc5c..949b5ab 100644
+--- a/Cargo.toml
++++ b/Cargo.toml
+@@ -32,6 +32,8 @@ async-channel = "1.8.0"
+ smol = "1.2"
+ futures-lite = "1.12.0"
+ micro-rdk = {version = "0.0.3", git = "https://github.com/viamrobotics/micro-rdk.git", features = ["esp32"]}
++micro-rdk-esp32-sensor-examples = { git = "https://github.com/viam-labs/micro-rdk-esp32-sensor-examples" }
+```
+
+Rebuild the project and reflash the board.
```

Comment:
usign the flasher or the dev instructions?

Reply:
Clarified that this is dev instructions.